image/Dockerfile

```diff
@@ -8,89 +7,83 @@ ENV GITREF=master
 ENV PRODUCT=uyuni
 ENV COMMAND=all-uyuni
 ENV SERVE=0
+ENV GEM_HOME="${REGULARUSER}/gems"
+ENV PATH=$GEM_HOME/bin:$GEM_HOME/gems/bin:$PATH
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 USER root
 RUN groupadd ${REGULARUSER} && \
-    useradd -d /home/${REGULARUSER} -m ${REGULARUSER}
+    useradd -d /home/${REGULARUSER} -m ${REGULARUSER} -g ${REGULARUSER}
 
 # All instructions extracted from
 # https://github.com/uyuni-project/uyuni-docs/wiki/Install-the-latest-documentation-toolchain
 
-# Base requirements
-RUN zypper -q -n ref && \
-    zypper -q -n in glibc-locale git gzip npm zip && \
-    zypper -q -n in -t pattern devel_C_C++ devel_python3 devel_perl
-
-# Install Ruby for building PDF’s
-RUN zypper -q -n in libyaml-devel sqlite3-devel readline-devel libopenssl-devel
+RUN zypper -q -n refresh
+RUN zypper -q -n update && zypper clean
+
+# Base requirements installable via patterns
```

Comment:
Since you' are doing this rewrite, I'd rather c onsidering consolidating all `zypper ref` then all `zypper in` and `zypper clean` on a single RUN call, so we get a single layer.